Sources/FoundationEssentials/ProcessInfo/ProcessInfo.swift

```diff
@@ -117,7 +117,11 @@ final class _ProcessInfo: Sendable {
     }
 
     var processIdentifier: Int32 {
-        return getpid()
+#if os(Windows)
+        return Int32(GetProcessId(GetCurrentProcess()))
```

Comment:
Yes it will trap if high bits are set. Is it possible for Windows to return a process ID with high bits set? (on Darwin `pid_t` is defined as a 32-bit signed integer). If so, is there a behavior here that you think would be better? (I believe in SCF it will trap today if the high bits are set)

Reply:
Yeah, it is. The ProcessId is a DWORD (UInt32). I guess we could do a `Int32(bitPattern:)` initializer?

Reply:
We could, but that would be a behavioral change from SCF today. Do you think that change / this new behavior would be expected for Windows clients of Foundation?

Reply:
I think that it would be more expected than a trap which would not make sense - I didn't create an `Int32`, why is this trapping?

Reply:
Ok I can update that to use a bit pattern then if you feel that'd be the better, more expected behavior for our windows clients